src/components/MediaPlayer/VideoJS/VideoJSPlayer.js

```diff
@@ -584,6 +584,18 @@ function VideoJSPlayer({
     return null;
   };
 
+  // Classes for setting caption size based on device
+  // Not all Android tablets return 'Android' in the useragent, so assume touch 
+  // devices are tablets. This should be safe because iPhones use the native player.
+  let videoClass = '';
+  if (IS_ANDROID) { 
+    videoClass = "video-js vjs-big-play-centered android";
+  } else if (IS_TOUCH_ONLY) {
+    videoClass = "video-js vjs-big-play-centered tablet"
+  } else { 
+    videoClass = "video-js vjs-big-play-centered"; 
+  };
```

Comment:
Thinking on it more, would it be better to do something like `else if (IS_TOUCH_ONLY && !IS_IOS) {...} else if (IS_IPAD) {...}` to explicitly exclude iPhones and iPods? Or should we not worry about it because of the native player overriding VideoJS on those devices?

Reply:
Might be good practice just to make sure in case we add more to those classes later.